radicle-cli/src/commands/remote.rs

```diff
@@ -0,0 +1,118 @@
+//! Remote Command implementation
+#[path = "remote/add.rs"]
+pub mod add;
+#[path = "remote/list.rs"]
+pub mod list;
+#[path = "remote/rm.rs"]
+pub mod rm;
+
+use std::ffi::OsString;
+
+use anyhow::anyhow;
+
+use radicle::prelude::Did;
+
+use crate::terminal as term;
+use crate::terminal::args::{Error, string};
+use crate::terminal::{Args, Context, Help};
+
+pub const HELP: Help = Help {
+    name: "remote",
+    description: "Manage set of tracked repositories",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+    rad remote
+    rad remote list
+    rad remote add <url>
+    rad remote rm <alias>
```

Comment:
Will `rad remote rm` be needed?  Will it do anything other than `git remote rm`?

Reply:
It should take an alias or a did; but regardless it makes sense to have it if we have `add`